scripts/UpdateRepos.ps1

```diff
@@ -105,7 +105,7 @@ try {
                 # Push the changes
                 if (-not $NoPush -and ($Force -or ($PSCmdlet.ShouldContinue("Pushing updates to repos.", 'Push the changes to these repos?')))) {
                     try {
-                        Invoke-Block { & git @gitConfigArgs push origin HEAD:$submodule.branch}
+                        Invoke-Block { & git @gitConfigArgs push origin HEAD:$(submodule.branch)}
```

Comment:
We're still iterating on this script, which pushes new packages back to the repositories.

Reply:
Syntax error. Should be `HEAD:$($submodule.branch)`